src/i18n/languages/fr-FR.js

```diff
@@ -0,0 +1,101 @@
+/**
+ * @preserve
+ * Authors: Stefan Salzl, Thomas Senn
+ * Last updated: Feb 05, 2018
+ *
+ * Description: Definition file for French - France language-country.
+ */
+import * as C from '../constants';
+
+const dictionary = {
+  languageCode: 'fr-FR',
+  [C.CONTEXTMENU_ITEMS_ROW_ABOVE]: 'Insérer une ligne en haut',
+  [C.CONTEXTMENU_ITEMS_ROW_BELOW]: 'Insérer une ligne en bas',
+  [C.CONTEXTMENU_ITEMS_INSERT_LEFT]: 'Insérer une colonne à gauche',
+  [C.CONTEXTMENU_ITEMS_INSERT_RIGHT]: 'Insérer une colonne à droite',
+  [C.CONTEXTMENU_ITEMS_REMOVE_ROW]: ['Supprimer une ligne', 'Supprimer les lignes'],
+  [C.CONTEXTMENU_ITEMS_REMOVE_COLUMN]: ['Supprimer une colonne', 'Supprimer les colonnes'],
+  [C.CONTEXTMENU_ITEMS_UNDO]: 'Annuler',
+  [C.CONTEXTMENU_ITEMS_REDO]: 'Répeter',

```

Comment:
A better translation of redo is 'Rétablir'

Reply:
Hi @buggythepirate can I count on your feedback?